packages/components/package.json

```diff
@@ -21,19 +21,15 @@
   ],
   "sideEffects": false,
   "exports": {
-    ".": {
-      "import": "./index.mjs",
-      "require": "./index.cjs"
-    },
+    ".": "./index.mjs",
     "./*": "./*"
   },
-  "main": "./index.cjs",
+  "main": "./index.mjs",
   "module": "./index.mjs",
   "unpkg": "./index.iife.min.js",
   "jsdelivr": "./index.iife.min.js",
-  "types": "./index.d.ts",
+  "types": "./index.d.mts",
   "files": [
-    "*.cjs",
     "*.d.cts",
```

Comment:
@userquin this?

Reply:
yes, also .d.cts and remove it from the rollup.config.ts file

Reply:
L122:

![imagen](https://github.com/user-attachments/assets/cd6c3421-4a03-4675-a321-9c873c5d677d)

Reply:
We should remove `.d.cts` from all files entries in all package.json files